### DISCUSSION TITLE
Windows 7

### DISCUSSION THREAD

**Discussion by bt4ibwem8** [cdgriffith/FastFlix#470] [General]
What components are missing in win7 in order to run Fastflix v5? I could try to install them manually so maybe also the new Fastflix version would be running in this system.

**Comment by cdgriffith** [comment]
The main GUI backend is QT6 which does not support Windows 7 https://doc.qt.io/qt-6/windows.html is the big blocker

**Comment by bt4ibwem8** [comment]
Thanks, i found these patched dlls so i can try it 😱
https://forum.qt.io/post/744065

**Comment by JamesBrooklyn1** [comment]
As far as I know, Fastflix v5 requires some components that are not available in Windows 7, so you might need to install them manually. I'm not exactly sure which components are missing, but you could try doing a quick search online or reaching out to the Fastflix community for more information.

**Reply by JasonJames3** [reply]
It's worth noting that upgrading to a newer version of Windows may improve the overall performance and compatibility of your programs.  So, you can check out trusted online communities for getting [windows 10 key cheap](https://www.reddit.com/r/cheapwindowskeys/comments/wjvsae/cheap_windows_keys/). It's essential to be careful when purchasing keys from third-party sellers, as some may be fraudulent or illegitimate.

**Reply by bt4ibwem8** [reply]
thanks but i do not like win10/win11... I will switch rather to linux when the death of win7 will come into my house :)
